manifests/base.pp

```diff
@@ -59,12 +66,19 @@
       false => {}
     }
 
+    $restart_service = $rsyslog::manage_service ? {
```

Comment:
I see no need to merge metaparameters like this.  It's much cleaner to conditionally add the relationships with [chaining arrows syntax](https://puppet.com/docs/puppet/7.3/lang_relationships.html#lang_rel_chaining_arrows).